compiler/rustc_smir/src/stable_mir/mod.rs

```diff
@@ -130,6 +131,8 @@ pub trait Context {
     /// Obtain the representation of a type.
     fn ty_kind(&mut self, ty: Ty) -> TyKind;
 
+    fn mk_const(&mut self, kind: TyKind) -> Ty;
```

Comment:
Can you explain a bit why this is called `mk_const` and not `mk_ty` or something?

Reply:
Lol what did I do